modules/shared/components/molecules/PostViewHeader/UserInfo/Index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Divider from '../../../atoms/Divider/Divider';
+import type { ReactElement, FC } from 'react';
+import VerticalThreeDotsIcon from '../../../icons/verticalThreeDots.svg';
+import type { PostViewHeaderTypes } from './PostViewHeaderTypes';
+import UserInfo from '../../UserInfo/Index';
+import * as EDivider from '../../../atoms/Divider/types/EDivider';
+
+const PostViewHeader: FC<PostViewHeaderTypes.IProps> = ({
+  size = 'medium',
+  variant,
+  imgSrc,
+  name,
+  date,
```

Comment:
which component responsible to parse/transform the date from iso formate to a human readable format like "2 days" or "3 hours"?

Reply:
we decided to parse the date in the parent

Reply:
I wouldn't suggest so. I would vote for the userinfo to receive date and parse it. I belive the parent will have too much responsibilities if we started to delegate everything to it.